web/packages/shared/components/CopyButton/CopyButton.tsx

```diff
@@ -31,13 +31,15 @@ export function CopyButton({
   value,
   mr,
   ml,
+  customTooltip,
 }: {
   value: string;
   mr?: number;
   ml?: number;
+  customTooltip?: string;
```

Comment:
IMHO `tooltip` as a name would be good enough – it's already "custom" through the fact of being supplied to the component.